include/aspect/simulator.h

```diff
@@ -804,7 +804,8 @@ namespace aspect
        */
       void denormalize_pressure(LinearAlgebra::BlockVector &vector);
 
-
+//FIXME apply the limiter to dg solutions
```

Comment:
Is this FIXME resolved? If not, and you think it should enter the main version, please change to TODO, this way we keep these notes more similar across the code.